pkg/crdconversion/crdconversion.go

```diff
@@ -96,25 +97,47 @@ func (crdWh *crdConversionWebhook) run(stop <-chan struct{}) {
 		}
 
 		// #nosec G402
-		server.TLSConfig = &tls.Config{
+		webhookServer.TLSConfig = &tls.Config{
 			Certificates: []tls.Certificate{cert},
 		}
 
-		if err := server.ListenAndServeTLS("", ""); err != nil {
+		if err := webhookServer.ListenAndServeTLS("", ""); err != nil {
 			log.Error().Err(err).Msg("crd-converter webhook HTTP server failed to start")
 			return
 		}
 	}()
 
+	healthMux := http.NewServeMux()
+
```

Comment:
nit: remove unnecessary newline

Reply:
I have another pr that will modify the same file, I will remove it then